scipy/stats/morestats.py

```diff
@@ -2428,7 +2432,7 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False):
     z = (T - mn - correction) / se
     prob = 2. * distributions.norm.sf(abs(z))
 
-    return WilcoxonResult(T, prob)
+    return WilcoxonResult(T, prob, rejection_direction)
```

Comment:
This introduces a backwards incompatibility. Is this discussed beforehand somewhere?